snuba/cli/migrate.py

```diff
@@ -3,33 +3,18 @@
 
 import click
 
-from snuba.datasets.factory import ACTIVE_DATASET_NAMES, get_dataset
 from snuba.environment import setup_logging
+from snuba.migrations.migrate import logger, run
 from snuba.util import local_dataset_mode
 
 
 @click.command()
 @click.option("--log-level", help="Logging level to use.")
-@click.option(
-    "--dataset",
-    "dataset_name",
-    type=click.Choice(ACTIVE_DATASET_NAMES),
-    help="The dataset to target",
-)
-def migrate(
-    *, log_level: Optional[str] = None, dataset_name: Optional[str] = None
-) -> None:
-    from snuba.migrations.migrate import logger, run
-
+def migrate(*, log_level: Optional[str] = None) -> None:
```

Comment:
Please ensure we do not pass the dataset name anywhere in our onpremise scripts or that such field is not documented anywhere

Reply:
Good call, I couldn't find any trace of it.